protocols/IFIN/enzyme_activity.ot1.py

```diff
@@ -0,0 +1,54 @@
+from opentrons import containers, instruments
+
+plate_a = containers.load('96-flat', 'A1')
+plate_b = containers.load('96-flat', 'B1')
+trash = containers.load('trash-box', 'A2')
+tiprack1 = containers.load('tiprack-200ul', 'B2')
+tiprack2 = containers.load('tiprack-200ul', 'C2')
+
+
+m200 = instruments.Pipette(
+    axis='a',
+    name='m200',
+    max_volume=200,
+    min_volume=20,
+    channels=8,
+    trash_container=trash,
+    tip_racks=[tiprack1, tiprack2]
+    )
+
+m200.pick_up_tip()
```

Comment:
Code is good, maybe some inline comments about which section is for which step in the process